airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/streams.py

```diff
@@ -232,6 +232,19 @@ class Chats(TimeIncrementalStream):
     data_field = "chats"
     limit = 1000
 
+    def next_page_token(self, response: requests.Response) -> Optional[Mapping[str, Any]]:
+        response_data = response.json()           
+        if response_data["count"] == self.limit:
+            obj = {
```

Comment:
```suggestion
            next_page = {
```

Reply:
Probably is better to give a more meaninful name or this variable.

Reply:
you are right!